atomic_reactor/inner.py

```diff
@@ -370,8 +370,9 @@ def build_docker_image(self):
                 raise
 
             self.builder.is_built = True
-            self.builder.image_id = self.build_result.image_id
-            self.built_image_inspect = self.builder.inspect_built_image()
+            if self.build_result.is_image_available():
+                self.builder.image_id = self.build_result.image_id
+                self.built_image_inspect = self.builder.inspect_built_image()
```

Comment:
There are quite a few plugins that read from these values, but none of them should actually be used with orchestrate_build plugin. Not sure what's the best way to handle plugin compatibility.